README.md

```diff
@@ -462,6 +468,61 @@ Connect to the database to configure and complete the following steps:
         ALTER DEFAULT PRIVILEGES FOR ROLE <privileged_user> GRANT SELECT, INSERT, UPDATE, DELETE ON TABLES TO base_role;
         ALTER DEFAULT PRIVILEGES FOR ROLE <privileged_user> GRANT ALL ON SEQUENCES TO base_role;
 
+# Email Settings
+SendGrid is the service used in this app to send automated email to users. Anything in this section references the `sendgrid` package, although a planned change will be to use [`django-anymail`](https://anymail.dev/en/stable) with the SendGrid option instead in order to genericize the email code.
+
+For the best user experience, the recommended email address to accept correspondence is the same address that automated emails are sent from. The code is set up this way, such that the `contact_email` variable is propagated throughout.
+
+# Phone Settings
+Twilio is the service used in this app to [send SMS](#sending-sms) and [receive voice calls](#call-forwarding).
+
+For the best user experience, the recommended number to accept calls is the same number that automated SMS are sent from. The code is set up this way, such that the `TWILIO_NUMBER` configuration variable is propagated throughout (prettified for display as `contact_number`). The [call-forwarding](#call-forwarding) section goes through the steps of setting up the SMS-sender number to forward voice calls to a real phone number.
+
+## Sending SMS
+Something here about how to set up Twilio to send SMS...
+
+## Call-Forwarding
+Voice calls from users are set up for simple pass-through forwarding. This is configured from the Twilio console.
+
+### Call-Forwarding Setup
+Twilio provides a simple page for initial setup, found at https://www.twilio.com/code-exchange/simple-call-forwarding under the default 'Quick Deploy to Twilio' tab. If the admin is already logged in to Twilio, all fields except 'My Phone Number' under Step 2 are prefilled. Fill in the number to forward to and select 'Deploy this application'.
+
+![Twilio Quick Deploy a forwarding number][1]
+
+To change the forwarding number, see [Call-Forwarding Configuration](#call-forwarding-configuration).
+
+## Call-Forwarding Configuration
+To reconfigure the voice forwarding function or change the target phone number, navigate to the function in the Twilio console.
+
+1. From the initial screen once logged in to Twilio, select 'Explore Products +'.
+
+1. Under the Developer Tools section, find 'Functions and Assets'. Select the pushpin icon to pin in to the Twilio sidebar navigation.
```

Comment:
Not that this is anything of substance... The configuration steps for call-forwarding all use '1.'